src/main/java/kg/orderservice/controller/OrderController.java

```diff
@@ -12,9 +14,17 @@
 public class OrderController {
```

Comment:
Үстүнкү сапты `/api/v1/order/delivery` деп өзгөртүп койсоң болот
```suggestion
public class DeliveryController {
```